.carve/ignore

```diff
@@ -0,0 +1,114 @@
+;; Used by shadow-cljs
+electron.core/main
+electron.core/start
+electron.core/stop
+;; repl fn
+electron.search/query
+frontend.blob/blob
+frontend.components.svg/arrow-left
+frontend.components.svg/arrow-right
+frontend.components.svg/big-arrow-right
+frontend.components.svg/big-arrow-left
+frontend.components.svg/arrow-narrow-left
+frontend.components.svg/arrow-narrow-right
+frontend.components.svg/minus
+frontend.components.svg/rectangle
+frontend.components.svg/user
+frontend.components.svg/plus
+frontend.components.svg/plus-circle
+frontend.components.svg/graph-sm
+frontend.components.svg/folder-add
+frontend.components.svg/folder-add-large
+frontend.components.svg/folder-sm
+frontend.components.svg/pages-sm
+frontend.components.svg/repos-sm
+frontend.components.svg/calendar-sm
+frontend.components.svg/import-sm
+frontend.components.svg/sort-asc-sm
+frontend.components.svg/vertical-dots
+frontend.components.svg/horizontal-dots
+frontend.components.svg/save
+frontend.components.svg/menu
+frontend.components.svg/excalidraw-logo
+frontend.components.svg/discord
+frontend.components.svg/slideshow
+frontend.components.svg/indent-block
+frontend.components.svg/outdent-block
+frontend.components.svg/move-up-block
+frontend.components.svg/move-down-block
+frontend.components.svg/multi-line-input
+frontend.components.svg/checkbox
+frontend.components.svg/online
+frontend.components.svg/collapse-right
+frontend.components.svg/git
+frontend.components.svg/home
+frontend.components.svg/icon-cmd
+frontend.components.svg/icon-editor
+frontend.components.svg/icon-cli
+frontend.components.svg/settings
+frontend.components.svg/arrow-expand
+;; Used by shadow-cljs
+frontend.core/stop
+;; Referenced in comment
+frontend.db.model/with-children-refs
+;; Referenced in comment
+frontend.db.model/get-latest-changed-pages
+;; For repl
+frontend.db.utils/q
+;; For debugging
+frontend.debug/defn
+frontend.debug/print
+;; Left as an example
+frontend.diff/diffs
+;; Lazily loaded
+frontend.extensions.code/editor
+;; Lazily loaded
+frontend.extensions.excalidraw/draw
+;; Referenced in commented TODO
+frontend.extensions.pdf.utils/get-page-bounding
+;; For repl
+frontend.extensions.zotero.api/item
+;; For repl
+frontend.external.roam/reset-state!
+;; For repl
+frontend.format.mldoc/ast-export-markdown
+;; Protocol fn wrapper that could be used
+frontend.fs/readdir
+;; Referenced in TODO
+frontend.handler.metadata/update-properties!
+;; Referenced in comment
+frontend.handler.route/toggle-between-page-and-file!
```

Comment:
This is referenced in commented out code in frontend.modules.shortcut.config. Should we keep or remove it?